source-academy/frontend
# 2748 remove hints type inference

### Description



Hints provided by type inferences will become redundant with the development of Source Typed.

### Type of change

- [x] Removed UseTypeInference hook from src/commons/editor
- [x] Removed typeCheck from src/sagas/WorkspaceSage.tsx

### How to test


Type hints will no longer be provided when type-incompatible operations are executed. The keyboard shortcut Ctrl-Shift-m will no longer show type information.

### Checklist





Closes #2748.
